cmd/flawfix/api/api.go

```diff
@@ -315,6 +316,8 @@ func Start(db core.DB) {
 	flawRouter.GET("/", flawController.ListPaged)
 	flawRouter.GET("/:flawId/", flawController.Read)
 
+	flawRouter.POST("/:flawId/", flawController.CreateEvent)
```

Comment:
We need to make sure, that the user is allowed to update the asset. We can do that easily by using the projectScopedRBAC:
```suggestion
	flawRouter.POST("/:flawId/", flawController.CreateEvent, projectScopedRBAC("asset", accesscontrol.ActionUpdate))
```